queries/update_table_metadata.py

```
@@ -0,0 +1,88 @@
+import json
+import os
+
+from google.cloud import bigquery
+
+
+def extrair_descricoes_de_modelos(manifest):
+    tabelas = []
+
+    for node_id, node in manifest.get("nodes", {}).items():
```

Ruff (0.12.2)

Loop control variable `node_id` not used within loop body

Rename unused `node_id` to `_node_id`

(B007)